plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveConfig.java

```diff
@@ -564,6 +565,19 @@ public HiveConfig setValidateBucketing(boolean validateBucketing)
         return this;
     }
 
+    public boolean isUsePartitionedBucketFunction()
+    {
+        return usePartitionedBucketFunction;
+    }
+
+    @Config("hive.use-partitioned-bucket-function")
```

Comment:
i am not sure about the toggle name.
What about something like "distribute writes" or "parallel partitioned bucketed inserts"?